buildkite/bazelci.py

```diff
@@ -765,21 +765,41 @@ def execute_commands(
         if not build_targets and not test_targets:
             raise BuildkiteException("There are neither build nor test targets")
 
+        include_json_profile = task_config.get("include_json_profile", [])
```

Comment:
FYI: if you enable this setting for a pipeline that is also part of the downstream pipeline, any downstream pipeline run will collect json files, too. I think this is fine.